Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -111,14 +114,16 @@ class TRDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
         labels.clear();
       }
     }
-
+    LOG(info) << " digits : " << digits.size() << " labels labelssize : " << labels.getNElements() << " labels index size : " << labels.getIndexedSize();
     // Force flush of the digits that remain in the digitizer cache
     mDigitizer.flush(digits, labels);
-    triggers.emplace_back(irecords[irecords.size() - 1], digitsAccum.size(), digits.size());
+    LOG(info) << " digits : " << digits.size() << " labels labelssize : " << labels.getNElements() << " labels index size : " << labels.getIndexedSize();
+    //triggers.emplace_back(irecords[irecords.size() - 1], digitsAccum.size(), digits.size());
```

Comment:
This part of the code requieres a cleanup, way too much printing left after debuting, maybe.

Reply:
its reverted to dev branch again.